enhancements/single-node-production-edge-cluster-profile.md

```diff
@@ -0,0 +1,380 @@
+
+A hypothetical DU example is an unusually resource-intensive workload,
+requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages,
+multiple SR-IOV NICs carrying several Gbps of traffic each, and
+specialized accelerator devices. The node hosting this workload must
+run a realtime
+kernel, be carefully tuned to ensure low-latency requirements
+can be met, and be configured to support features like Precision Timing
+Protocol (PTP).
+
+One crucial detail of this use case is the "cloud" hosting this workload
+is expected to be "autonomous" such that it can continue operating with
+its existing configuration and running the existing workload, even when
+any centralized management functionality is unavailable.
+
+### Goals
+
+* This enhancement describes an approach for deploying OpenShift in
+  single-node configurations for production use in environments with
+  "reasonably significant" memory, storage, and compute resources.
+* Clusters built using the `single-node-production-edge` profile
+  should pass most Kubernetes and OpenShift conformance end-to-end
+  tests. Any tests that must be skipped due to differences from a
+  multi-node deployment will be documented.
+
+### Non-Goals
+
+* This enhancement does not address single-node deployments in
+  highly-constrained environments such as Internet-of-things devices
+  or personal computers.
+* This enhancement does not address "developer" use cases. See the
+  [single-node-developer-profile](https://github.com/openshift/enhancements/pull/302)
+  enhancement.
+* This enhancement does not address high-availability for single-node
+  deployments.
+* This enhancement does not address in-place upgrades for this first
+  iteration. Upgrades will initially only be achieved by redeploying
+  the machine and its workload.
+* This enhancement does not attempt to describe a way to "pre-build"
+  deployment images, either generically or customized for a user.
+* This enhancement does not address the removal of the bootstrap VM,
+  although single-node clusters would benefit from that work, which
+  will be described in a separate enhancement.
+
+## Proposal
+
+After the profile is introduced, all teams developing OpenShift
+components will need to consider how their components should be
+configured when deployed and used in the `single-node-production-edge`
+deployments.
+
+Although the environment is assumed to have significant resources, it
+is important to dedicate most of them to end-user workloads, rather
+than cluster control plane or monitoring. Therefore, the cluster
+profile will configure telemetry and logging to forward data, instead
+of collecting it locally.
+
+Edge deployments are typically part of a large fleet that is managed
+automatically rather than one at a time. Therefore the console will
+not be deployed by this profile.
+
+The profile describes single-node, all-in-one, deployments, so there
+is no need to support provisioning additional workers. The
```

Comment:
A Telco requirement would be to add worker nodes to an all-in-one deployment

Reply:
We're specifically calling that out as something we would not do with these types of clusters. What sort of use case would call for a single-node control plane with separate worker(s)?

Reply:
I believe if there was need for additional compute resources at a far edge location, the telco would be more likely to deploy a second independent single-node cluster rather than adding an additional worker to the existing single-node cluster so that there uniformity in implementation across their network.